StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -112,20 +110,14 @@ extension SavedPaymentMethodCollectionView {
 
         var cbcEligible: Bool = false
         var allowsPaymentMethodRemoval: Bool = true
-        var alternateUpdatePaymentMethodNavigation: Bool = false
-
-        /// Indicates whether the cell should be editable or just removable.
-        /// If the card is a co-branded card and the merchant is eligible for card brand choice, then
-        /// the cell should be editable. Otherwise, it should be just removable.
-        var shouldAllowEditing: Bool {
-            if alternateUpdatePaymentMethodNavigation {
-                return UpdatePaymentMethodViewModel.supportedPaymentMethods.contains { type in
-                    viewModel?.savedPaymentMethod?.type == type
-                }
-            }
-            else {
-                return (viewModel?.isCoBrandedCard ?? false) && cbcEligible
+
+        /// Indicates whether the cell for a saved payment method should display the edit icon.
+        /// True if payment methods can be removed or edited (will update this to include allowing set as default)
+        var showEditIcon: Bool {
+            guard UpdatePaymentMethodViewModel.supportedPaymentMethods.contains(where: { viewModel?.savedPaymentMethod?.type == $0 }) else {
+                fatalError("Payment method does not match supported saved payment methods.")
             }
+            return allowsPaymentMethodRemoval || (viewModel?.savedPaymentMethod?.isCoBrandedCard ?? false && cbcEligible)
```

Comment:
Since all cards are now editable, do we need " (viewModel?.savedPaymentMethod?.isCoBrandedCard ?? false && cbcEligible)"?

Reply:
Yep that sounds right.